build.sbt

```diff
@@ -37,3 +37,15 @@ lazy val http4s = project
   .dependsOn(cats)
   .settings(name := "memeid-http4s")
   .settings(dependencies.common, dependencies.http4s)
+
+lazy val docs = project
```

Comment:
You don't need to create a project for docs, just add the `MdocPlugin` to the root one:

```diff
lazy val root = project
  .in(file("."))
  .settings(name := "memeid")
+ .enablePlugins(MdocPlugin)
- .aggregate(core, cats, literal, doobie, circe)
+ .aggregate(core, cats, literal, doobie, circe, http4s)
+ .dependsOn(core, cats, literal, doobie, circe, http4s)
  .settings(skip in publish := true)
+ .settings(dependencies.docs)
+ .settings(mdocVariables := Map("VERSION" -> version.value))
```